server/app.js

```diff
@@ -6,7 +6,9 @@ const cookieParser = require('cookie-parser');
 const bodyParser = require('body-parser');
```

Comment:
Spaces between parens, params, and function calls. Please remove all semi-colons